runtime/commonTest/src/kotlinx/serialization/json/JsonTransformingSerializerTest.kt

```diff
@@ -35,7 +35,7 @@ class JsonTransformingSerializerTest : JsonTestBase() {
 
     object DroppingNameSerializer : JsonTransformingSerializer<Example>(Example.serializer(), "DropName") {
         override fun writeTransform(element: JsonElement): JsonElement =
-            JsonObject(element.jsonObject.filterNot { (k, v) -> k == "name" && v.primitive.content == "First" })
+            JsonObject(element.jsonObject.filterNot { (k, v) -> k == "name" && v.jsonPrimitive.content == "First" })
```

Comment:
AFAIR the same example was in docs/json_transformations.md, fix it there too, please